File: Java/src/pagina74b.java
import java.util.Scanner;

public class pagina74b {

    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);
        int[] A = new int[8];
        int[] B = new int[8];

        for (int i = 0; i < 8; i++) {
            System.out.println("Adicione um número ao vetor A: ");
            A[i] = scanner.nextInt();
            B[i] = A[i] * 3;
        }
        scanner.close();

        for (int i = 0; i < 8; i++) {
            System.out.print(B[i] + " ");
        }

    }

}

File: JavaScript/Manzano/Pagina74b.js
// Programa que lê 8 elementos de uma matriz tipo vetor A e calcula a matriz B, onde cada elemento de B é o triplo do elemento correspondente em A.

const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

const A = [];
const B = [];

function lerElementoVetor() {
  rl.question("Adicione um número ao vetor A: ", function (input) {
    const numero = parseInt(input);
    A.push(numero);
    B.push(numero * 3);

    if (A.length < 8) {
      lerElementoVetor();
    } else {
      rl.close();
      apresentarVetorB();
    }
  });
}

function apresentarVetorB() {
  for (let i = 0; i < B.length; i++) {
    process.stdout.write(B[i] + " ");
  }
}

lerElementoVetor();